src/components/encryption.ts

```diff
@@ -86,16 +86,18 @@ export class EncryptedEventBroker {
         private onEphemeralEvent: ((event: EphemeralEvent) => void)|undefined,
         private getIntent: (userId: string) => Intent
         ) {
+        if (this.onEphemeralEvent) {
```

Comment:
If we're not handling presence, we don't need a cleanup timer.

Reply:
this could be a comment in the code, not here